example_scenes/basic.py

```diff
@@ -236,3 +237,24 @@ def construct(self):
         )
         self.play(Write(on_screen_subscript_var))
         self.wait()
+
+
+class UnitDot(Scene):
+    def construct(self):
+        path = VMobject()
+        dot = Dot()
+        path.set_points_as_corners([dot.get_center(), dot.get_center()])
+
+        def update_path(path):
+            previus_path = path.copy()
+            previus_path.add_points_as_corners([dot.get_center()])
+            path.become(previus_path)
+
+        path.add_updater(update_path)
+        self.add(path, dot)
+        self.play(Rotating(dot, radians=PI, about_point=RIGHT, run_time=2))
+        self.wait()
+
+        self.play(dot.shift, PercentUnit(25 * UP))
+        self.play(dot.shift, PixelUnit(100 * LEFT))
```

Comment:
I like that *Unit classes can accept vectors and not only single units. Can we add another example where a *Unit class accepts a single value and not a vector too, for completeness?

@kolibril13 do you think this example should go in the example_scenes file or in the examples chapter of the docs?

Reply:
Yeah, I'll take a look at that too, shift only takes vectors though so I'll test that on another method.

This "example" needs to be cleaned up as well as it contains stuff that is not really relevant for demonstrating units.

Reply:
Yep, I think they would be nice to be in the section:
https://manimce.readthedocs.io/en/latest/examples/shapes.html
And maybe we can rename this to "Shapes, Images and Positions".
An example that I made is the following:
```py
class UnitDot(Scene):
    def construct(self):

        dot1 = Dot(radius=0.2).to_edge(DOWN)
        dot2 = Dot(radius=0.2).to_edge(DOWN).shift(RIGHT)
        dot3 = Dot(radius=0.2).to_edge(DOWN).shift(2*RIGHT)
        self.add(dot1,dot2,dot3)
        self.play(dot1.shift, PercentUnit(1 * UP))
        self.play(dot2.shift, PercentUnit(0.5 * UP))
        self.play(dot3.shift, PixelUnit(400 * UP))
        self.play(dot3.shift, PixelUnit(100 * RIGHT))
        self.wait()
```
Note that this is now highly rendering quality dependent!
PercentUnit behaves differently with the -l flag and the -m flag.

Reply:
Alright, that might be a bug, I'll look into it!

Reply:
It indeed appears that it is being shifted correctly using your code. Do you know of a way to add a grid to the scene? That way we'd have visual guidelines to confirm that.

Reply:
AFAIK, there is currently no way of adding guidelines. Much needed feature.

Reply:
@kolibril13 found what was causing this issue. Will resolve it.

Reply:
@jvdoorn: Nice, sounds great!